packages/gatsby-theme-blog/gatsby-browser.js

```diff
@@ -0,0 +1,8 @@
+export const onRouteUpdate = ({ prevLocation }) => {
+     if (prevLocation !== null) {
+       const skipLink = document.querySelector("[data-reach-skip-link]")
+       if (skipLink) {
+         skipLink.focus()
```

Comment:
Would this make the skip link visible after each navigation change?

Reply:
This sets the user's keyboard focus to the skipnav link when they access each page. It's based on Madalyn's example repo and appears to be best practices, but I do wonder if that overrides something like an anchor tag nav on a blog post 🤔